src/System.Diagnostics.FileVersionInfo/src/Interop/Interop.Windows.cs

```diff
@@ -37,7 +37,7 @@ internal partial class mincore
     internal enum Constants : uint
     {
         ErrorSuccess = 0x0u,
-        FileVerGetLocalised = 0x1u,
+        FileVerGetLocalized = 0x1u,
```

Comment:
This is the spelling of the corresponding Windows constant. I think it would be better to leave it.

https://msdn.microsoft.com/en-us/library/windows/desktop/aa969434(v=vs.85).aspx